responses/__init__.py

```diff
@@ -202,6 +202,8 @@ def get_wrapped(
         Wrapped function
 
     """
+    if assert_all_requests_are_fired is None:
+        assert_all_requests_are_fired = mock.assert_all_requests_are_fired
```

Comment:
that makes the method dependent on the global variable `mock`. If someone downstream relies on this method, then it will get broken.

will that work?
```suggestion
        assert_all_requests_are_fired = responses.assert_all_requests_are_fired
```

Reply:
Changed.